cfpb/cfgov-refresh
# Add order field to Browse(Filterable)Page to order side nav

The secondary navigation requires the ability to order the links in whichever way the content editor wants. This adds a field to take care of the ordering. The default is 1.

## Additions

- `secondary_nav_order` field on Browse(Filterable)Page's

## Testing

- run `./cfgov/manage.py migrate`
- go to browse-type pages and set the secondary nav order field to be the number in which you'd like it to show in the secondary navigation. (Defaulted to 1)
- share/publish the page and verify the order is what you've set

## Review

- @kave 
- @richaagarwal
